src/Entity/ArticleCategory.php

```diff
@@ -110,6 +135,30 @@ public function setSlug(?string $slug): self
         return $this;
     }
 
+    public function getCtaLink(): ?string
+    {
+        return $this->ctaLink;
+    }
+
+    public function setCtaLink(?string $ctaLink): self
+    {
+        $this->ctaLink = $ctaLink;
+
+        return $this;
```

Comment:
We are not using fluent interface, please remove this.

Reply:
Are you sure ?all setter in this class are fluent :s

Reply:
Then it doesn't follow the conventions of this project.
if it's useless it should be refactored then (out of scope of your PR)

Reply:
You are right, I did not know.
My bad 👍

Reply:
@B-Galati can you approve this PR ? 💃